pages/dao/[symbol]/proposal/[pk]/index.tsx

```diff
@@ -131,6 +133,7 @@ const Proposal = () => {
       </div>
 
       <div className="col-span-12 md:col-span-5 lg:col-span-4 space-y-4">
+        {isMulti ? <VotePanel />: ""}
```

Comment:
Let's just move `<VotePanel />` here in all cases, regardless of multi vote. Dan's designs for normal votes put it first, too.